server/plugins/logger.js

```diff
@@ -3,88 +3,33 @@ module.exports.register.attributes = {
   name: 'hoodie-local-logger'
 }
 
-var qs = require('querystring')
-
-var log = require('npmlog')
-var Squeeze = require('good-squeeze').Squeeze
-
-function GoodEvent (events, config) {
-  this._callback = config.callback || function () {}
-  this._filter = new Squeeze(events)
-}
-
-GoodEvent.prototype.init = function (readstream, emitter, callback) {
-  readstream.pipe(this._filter).on('data', this._callback)
-  callback()
-}
-
 function register (server, options, next) {
   server.register({
     register: require('good'),
     options: {
-      opsInterval: 30000,
-      reporters: [{
-        reporter: GoodEvent,
-        config: {
-          callback: logger
-        },
-        events: {
-          ops: '*',
-          response: '*',
-          log: '*',
-          error: '*',
-          request: '*',
-          wreck: '*'
-        }
-      }]
+      ops: {
+        interval: 30000
+      },
+      reporters: {
+        hoodieReporter: [
+          {
+            module: 'good-squeeze',
+            name: 'Squeeze',
+            args: [
+              {
+                ops: '*',
+                response: '*',
+                log: '*',
+                error: '*',
+                request: '*'
+              }
+            ]
+          },
+          {
+            module: '../../../server/plugins/good-hoodie'
```

Comment:
Unfortunately, the file isn't being `require`d here.  Instead, that's being done somewhere inside of good.  So the path needs to be relative to where the good file is located.

I recommend that we change it to either `path.join(__dirname, 'good-hoodie')` or `path,join(process.cwd(), 'good-hoodie')`, in case the folks at hapi decide to change the location of the file in question. 

Another option is to create a new repository and then we could reference it without a relative path, like `module: '@hoodie/good-hoodie'`, or something like that.